### PR TITLE
[15.0][MIG] product_lot_sequence

### PR DESCRIPTION
**Pull request by i-vyshnevska**
plus include
https://github.com/OCA/product-attribute/pull/1111

### CODE REVIEW

**Comment by grindtildeath** [comment]
@i-vyshnevska Can you please check https://github.com/camptocamp/product-attribute/pull/2 ?

**Review by grindtildeath — APPROVED** [review]

**Comment by simahawk** [comment]
@i-vyshnevska pls remove the merge commit

**Review by sebalix — APPROVED** [review]

File: product_lot_sequence/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Adria Gil Sorribes <adria.gil@forgeflow.com>
+* Domantas Girdžiūnas <domantas@vialaurea.lt>
```
Comment (simahawk) [inline product_lot_sequence/readme/CONTRIBUTORS.rst:2]: you can add yourselves here

**Review by simahawk — CHANGES_REQUESTED** [review]
history to fix

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**